src/agent/libclusterfuzz/third-party/build.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+
+import string
+import constants
+
+data = {}
+
+name_check = string.ascii_uppercase + "_"
+
+HEADER = """
+// Rust generated by Microsoft, code originally from:
+//      https://github.com/google/clusterfuzz/blob/master/src/
+//          python/lib/clusterfuzz/stacktraces/constants.py
+//
+// Original Copyright:
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+"""
+
+for name in sorted(dir(constants)):
```

Comment:
The `dir(constants)` pattern makes me worry that it might be a little brittle. Can we just document any relevant assumptions or gotchas here?

Reply:
Sorting before insertion like this makes sense, but because we are on Python 3.7+, and so iteration over the plain old `dict` we accumulate will be guaranteed to be in insertion order. This is subtle enough to deserve a comment, IMO.